core/common/src/main/java/alluxio/conf/Configuration.java

```diff
@@ -359,6 +373,30 @@ public static InstancedConfiguration copyGlobal() {
     return new InstancedConfiguration(SERVER_CONFIG_REFERENCE.get().copyProperties());
   }
 
+  /**
+   * Gets all configuration properties filtered by the specified scope.
+   *
+   * @param scope the scope to filter by
+   * @return the properties
+   */
+  public static List<ConfigProperty> getConfiguration(Scope scope) {
```

Comment:
The only logical change in this PR is removing `AlluxioConfiguration` from this method. All callers use `Configuration.global()` so it's not necessary.